src/shared/install.c

```diff
@@ -861,13 +864,19 @@ static int find_symlinks_in_directory(
                 bool match_name,
                 bool ignore_same_name,
                 const char *config_path,
-                bool *same_name_link) {
+                bool *ret_same_name_link) {
```

Comment:
This is only updated when the names are same. So, cannot use our usual naming style.